packages/cloud-cognitive/src/components/EditTearsheetNarrow/EditTearsheetNarrow.stories.js

```diff
@@ -0,0 +1,293 @@
+/**
+ * Copyright IBM Corp. 2022, 2022
+ *
+ * This source code is licensed under the Apache-2.0 license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import React, { useState } from 'react';
+// TODO: import action to handle events if required.
+import { action } from '@storybook/addon-actions';
+import {
+  Button,
+  Dropdown,
+  NumberInput,
+  TextInput,
+  FormGroup,
+} from 'carbon-components-react';
+import { pkg } from '../../settings';
+import {
+  getStoryTitle,
+  prepareStory,
+} from '../../global/js/utils/story-helper';
+
+import { EditTearsheetNarrow } from '.';
+import { CreateTearsheetNarrow } from '../CreateTearsheetNarrow';
+import mdx from './EditTearsheetNarrow.mdx';
+
+import styles from '../CreateTearsheetNarrow/_storybook-styles.scss';
+
+export default {
+  title: getStoryTitle(EditTearsheetNarrow.displayName),
+  component: CreateTearsheetNarrow,
+  parameters: {
+    styles,
+    docs: {
+      page: mdx,
+    },
+  },
+};
+
+const createTearsheetNarrowBlockClass = `${pkg.prefix}--create-tearsheet-narrow--story`;
+
+const defaultStoryProps = {
+  title: 'Edit partition',
+  className: 'test-class-name',
+  description: 'Select the number of partitions you want to create',
+  formTitle: 'Core configuration',
+  formDescription:
+    'We recommend you fill out and evaluate these details at a minimum before deploying your topic.',
+  primaryButtonText: 'Save',
+  secondaryButtonText: 'Cancel',
+  label: 'Test label',
+  selectorPrimaryFocus: '#tearsheet-narrow-story-text-input--1',
+};
+
+const Template = (args) => {
+  const [open, setOpen] = useState(false);
+  const [topicName, setTopicName] = useState('Enter topic name here');
+  const [partitionCount, setPartitionCount] = useState(1);
+  const [replicaCount, setReplicaCount] = useState(1);
+  const [minimumInSyncReplicaCount, setMinimumInSyncReplicaCount] = useState(1);
+  const [retentionTime, setRetentionTime] = useState(1);
+  const [quantity, setQuantity] = useState(1);
+  const [items] = useState(['Day(s)', 'Month(s)', 'Year(s']);
+  const numberInputsInvalid =
+    partitionCount <= 0 ||
+    replicaCount <= 0 ||
+    minimumInSyncReplicaCount <= 0 ||
+    retentionTime <= 0 ||
+    quantity <= 0;
+  return (
+    <div>
+      <style>{`.${defaultStoryProps.className} { opacity: 0 }`};</style>
+      <Button onClick={() => setOpen(!open)}>
+        {open ? 'Close EditTearsheetNarrow' : 'Open EditTearsheetNarrow'}
+      </Button>
+      <CreateTearsheetNarrow
```

Comment:
i think i'm confused about what's going on here. you're not actually rendering `<EditTearsheerNarrow />` anywhere?